ansible/roles/integration-tests/tasks/community_pipeline.yaml

```diff
@@ -0,0 +1,94 @@
+---
+- name: Test the Community Pipeline
+  tags:
+    - test-community-hosted-pipeline
+  block:
+    - name: Start the Community Pipeline
```

Comment:
I think this can be dropped since the PR opening triggers the pipeline.

Reply:
Yes you're right, I planned to do it as final clean up